src/components/accounts/SelectedUtxosHeader.tsx

```diff
@@ -0,0 +1,156 @@
+import { StyleSheet, View, TouchableOpacity } from "react-native";
+
+import { NavigationProp } from "@react-navigation/native";
+
+import { useAccountsContext } from "./AccountsContext";
+import { useTransactionBuilderContext } from "./TransactionBuilderContext";
+import { AppText } from "../shared/AppText";
+import { Sats } from "./Sats";
+import { Colors, Typography } from "../../styles";
+import { Utxo } from "../../models/Utxo";
+import BubblesIcon from '../../assets/images/bubbles.svg';
+import ListIcon from '../../assets/images/list.svg';
+
+interface Props {
+  toggleScreenAction: 'bubbles' | 'list',
```

Comment:
this could be a an enum type or create a const object with these values so we don't have to explicitly type "bubbles"  in different screens and increase chances for typo errors.

Reply:
I've thought this over.  The string literal union type should be as typesafe as an enum.  When you start typing the string the IDE should autocomplete the options.  In my testing if I set a value not in the literal union the IDE typescript service picks it up and complains.

Also interestingly neither the enum or literal union seem to be completely typesafe.  While the IDE flags it, I can still set values not in them and the app still runs without compile or runtime error.

The main reason I prefer the string literal union in this case is I don't need to expose an enum.  This header will be used from two screens only.  We could create a HeaderScreenActions enum as you mentioned above, but I wouldn't want to use it for actions that appear on multiple different screens - would want it confined to just actions of this component.

Reply:
I was talking about string enums:

```
enum HeaderScreenActions {
 BUBBLES = 'bubbles',
 LIST = 'list
}
```

We are fine with string literal unions as well, if the usage is limited to one or two screens only.  
I guess we have to update the tsconfig file to avoid ignoring such errors during compilation.